README.md

```diff
@@ -137,3 +137,13 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
 `python ACCERT/src/Main.py -i myinput.son`
 
 
+
+# A Tool for Quantifying Capital Cost Reduction Reduction Pathways for Advanced Nuclear Reactors
+- A new tool has been developed to simulate the different pathways to reduce the costs of advanced nuclear power plants when constructed through large order books. This tool inputs various levers that represent high-level decision making and outputs the capital costs and construction duration starting from the first plant to the last plant in the order book.
+
+-The methodology and assumptions behind this tool are documented [here]: https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
```

Comment:
Looks like there's a typo here..the hyperlink may not show up

Reply:
I just fixed the typos. Also, the hyperlink will work. I can see that it is working on on ACCERT Repo:
https://github.com/BotrosHanna-INL/ACCERT